### PR TITLE
Adds inviteMemberEmailAddresses method to sdk.

### PR DESCRIPTION
**Pull request by knappe**

### CODE REVIEW

**Comment by randy-girard** [comment]
Docs: :white_check_mark: , Tests: :white_check_mark:. Needs a rebase, but looks good to me. :+1:

**Comment by knappe** [comment]
Hey @matthopson you stole my version number!  Mind taking a look?

File: docs/collections/member_email_addresses.md [hunk 1]
```diff
@@ -115,3 +116,41 @@ teamsnap.saveMemberEmailAddress(memberEmailAddress).then(function(){
...
+  * `memberEmailAddressIds`: [string] A comma separated list of ids
+  * `teamId`: [int] Id of Team
+  * `memberId`: [int] Id of member associated with `memberEmailAddressIds`
+  * `notifyAsMemberId`: [int] Id of member sending invitation
+  * `introduction`: [string] Introduction text for invitation (optional)
+
+### Examples
+```javascript
+// ~~~~~
+// Deletes a memberEmailAddress item.
```
Comment (matthopson) [inline docs/collections/member_email_addresses.md]: Comment doesn't match

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # TeamSnap JavaScript SDK CHANGELOG
 
+### Feb 8, 2016 // Version 1.9.0
```
Comment (matthopson) [inline CHANGELOG.md]: Missing changelog entry from prev PR (assuming that's the merge conflict)

**Comment by matthopson** [comment]
@knappe Impossible! Mine was a patch! :D

**Comment by knappe** [comment]
@matthopson How about now?  I've brought in your changes (thanks for the merge ;P).

**Comment by matthopson** [comment]
Don't forget `brunch build --production` and then :+1:

**Comment by knappe** [comment]
Will do!  I probably shouldn't have version bumped -- I think the changes are a little ways off for web and, subsequently, the sdk.  But now I know what a good PR looks like for the sdk. :)

**Comment by knappe** [comment]
@matthopson A slight change.  We added in the `INVITED_STATES` to the SDK and I also locked down our dependencies.  Thoughts?

**Comment by matthopson** [comment]
@knappe We should note the dep update in the CHANGELOG, I think.

File: src/collections/memberEmailAddresses.coffee [hunk 3]
```diff
@@ -14,6 +15,21 @@ exports.createMemberEmailAddress = (data) ->
     type: 'memberEmailAddress'
     receivesTeamEmails: true
 
+exports.inviteMemberEmailAddresses = (options, callback) ->
+  unless options.memberEmailAddressIds
+    throw new TSArgsError 'teamsnap.inviteMemberEmailAddresses', 'options.memberEmailAddressIds is required.'
```
Comment (matthopson) [inline src/collections/memberEmailAddresses.coffee]: Are we breaking the line-length on these?

**Comment by matthopson** [comment]
Couple of small nits and we can button this up.

**Comment by matthopson** [comment]
Thanks! :+1:

**Pull request merged**